client.lua

```diff
@@ -926,7 +925,7 @@ exports("GetGlobalTypeData", function(type, label) return Types[type][label] end
 
 exports("GetZoneData", function(name) return Zones[name] end)
 
-exports("GetTargetBoneData", function(bone) return Bones.Options[bone][label] end)
+exports("GetTargetBoneData", function(bone) return Bones.Options[bone] end)
```

Comment:
Instead of removing label, add it to the function parameters as I forgot to do that

Reply:
Alright. I thought it was meant to get the whole bone options